tests/providers/celery/executors/test_celery_kubernetes_executor.py

```diff
@@ -49,6 +49,9 @@ def test_serve_logs_default_value(self):
     def test_is_single_threaded_default_value(self):
         assert not CeleryKubernetesExecutor.is_single_threaded
 
+    def test_no_cli_commands_vended(self):
```

Comment:
Should it be here and in others 'test_cli_commands_vended` instead ?

Reply:
Yupp, the combination executors used to not vend commands, but I changed it to vend after a comment from Jed. I'll update the name here :+1: